RevenueCatUI/CustomerCenter/Abstractions/CustomerCenterPurchasesType.swift

```diff
@@ -14,11 +14,13 @@
 import Foundation
 import RevenueCat
 
+// swiftlint:disable missing_docs
+
 @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
 @available(macOS, unavailable)
 @available(tvOS, unavailable)
 @available(watchOS, unavailable)
-protocol CustomerCenterPurchasesType: Sendable {
+@_spi(Internal) public protocol CustomerCenterPurchasesType: Sendable {
```

Comment:
Hmm I'm thinking we could potentially avoid exposing this, and in the `CustomerCenterView` constructor, just expose the parameters we want to override/mock as parameters right? Not sure if we do need to mock anything from our app instead.

Reply:
> Not sure if we do need to mock anything from our app instead.

This is exactly why I opened up `CustomerCenterPurchasesType`. It does contain not only parameters, but also methods. In fact, it contains all the `Purchases` interfaces that the Customer Center needs to function. So as you say, the app needs to be intercepting these interfaces (parameters but also methods) to be able to mock data for Customer Center previews.

Does it make sense?

Reply:
Ahh right. Yeah makes total sense. Thanks for explaining!